build.sh

```diff
@@ -23,7 +23,7 @@ sudo apt-get install -y \
 
 sudo mkdir /usr/local/php7
 
-git clone https://github.com/php/php-src.git
+git clone -b PHP-7.0.0 --depth=1 https://github.com/php/php-src.git
 cd php-src
 git checkout PHP-7.0.0
```

Comment:
Doing ` --depth=1` would require deleting the local repo whenever the version number is updated in the build script. A workaround is to use `--no-single-branch` but that results in a 108.04 MiB transfer instead of 19.47 MiB (for single branch). Or we could delete the existing `php-src` before every build.